code/obj/item/mops_cleaners.dm

```diff
@@ -1192,3 +1194,99 @@ TYPEINFO(/obj/item/handheld_vacuum/overcharged)
 		animate(alpha=0, time=0.21 SECONDS, easing=SINE_EASING)
 		SPAWN(0.5 SECONDS)
 			qdel(src)
+
+
+/obj/item/trash_bag
+	name = "trash bag"
+	desc = "A flimsy bag for filling with things that are no longer wanted."
+	icon = 'icons/obj/janitor.dmi'
+	inhand_image_icon = 'icons/mob/inhand/hand_tools.dmi'
+	icon_state = "trashbag-f"
+	item_state = "trashbag-f"
+	uses_multiple_icon_states = 1
+	w_class = W_CLASS_TINY
+	rand_pos = 1
+	flags = FPRINT | TABLEPASS | NOSPLASH
+	tooltip_flags = REBUILD_DIST
+	var/base_state = "trashbag"
+
+	New()
+		..()
+		src.create_storage(/datum/storage/no_hud, prevent_holding = list(/obj/item/trash_bag), max_wclass = W_CLASS_NORMAL, slots = 20,
+			params = list("use_inventory_counter" = TRUE, "variable_weight" = TRUE, "max_weight" = 20))
+
+	equipped(mob/user)
+		..()
+		for (var/i = 1 to round(length(src.storage.get_contents()) / 3))
+			src.remove_random_item(user)
+
```

Comment:
```suggestion
```
Could probably remove this code? Was used for when equipping the trashbag on your back, but seems it can't be anymore

Reply:
I was 100% sure I removed and committed this as you say it's no longer relevant.